python/HeaderService/camera_coords.py

```diff
@@ -57,13 +57,33 @@
                     'dimh': 509,
                     'preh': 3,
                     'overh': 32,
-                    'overv': 48}
+                    'overv': 48,
+                    'ccdax': 4000,
+                    'ccday': 4072,
+                    'ccdpx': 4198,
+                    'ccdpy': 4198,
+                    'gap_inx': 27,
+                    'gap_iny': 27,
+                    'gap_outx': 26,
+                    'gap_outy': 26,
+                    'raftx': 12700,
+                    'rafty': 12700}
 
 SCAN_GEOM['E2V'] = {'dimv': 2002,
                     'dimh': 512,
                     'preh': 10,
                     'overh': 54,
-                    'overv': 46}
+                    'overv': 46,
+                    'ccdax': 4004,
+                    'ccday': 4096,
+                    'ccdpx': 4197,
+                    'ccdpy': 4200,
+                    'gap_inx': 28,
+                    'gap_iny': 25,
+                    'gap_outx': 26.5,
+                    'gap_outy': 26,
+                    'raftx': 12700,
+                    'rafty': 12700}
 
```

Comment:
Why are these numbers hard-coded here, and why are there numbers for both e2v and itl devices (you just argued that the CCD serial number isn't going to change).  I am not happy that any of these numbers are in the header service -- LSST gets them from the cameraGeometry and they are not reliable even for the telescope as the number of overscan pixels can change when the sequencer files are changed,

Reply:
I added them to have access to these definitions in the future. If you want them out, I can removed them.

Reply:
I'd like all non-essential code removed from this changeset, please.  In other words, a minimal changeset that adds a minimal WCS.